### DISCUSSION TITLE
Adafruit Itsybitsy M4 with PJRC Display

### DISCUSSION THREAD

**Discussion by JoshBeckmann** [ImpulseAdventure/GUIslice#346] [Q&A] [answered]
My programming skills are pretty basic. Thank you for your generous tool and patience!

I'm using the Adafruit Itsybitsy M4 with a PJRC Display. I have the display/touch working quite well with the MCU. I'm following your configuration check list. When I run the basic confirmation diagnostics (ex01_ard_basic) I get the following error: 

`C:\Users\workstation\Documents\Arduino\libraries\GUIslice\src\GUIslice.c:62:14: fatal error: pgmspace.h: No such file or directory
   62 |     #include <pgmspace.h>
      |              ^~~~~~~~~~~~
compilation terminated.
exit status 1
Error compiling for board Adafruit ItsyBitsy M4 (SAMD51).
`

Can you offer some assistance? Thanks again! 

•	Identify the display and the appropriate graphics & touch drivers for it.
_PJRC 
Adafruit ItsyBitsy M4_

•	Confirm the stand-alone graphics and touch library tests work (ie. Adafruit graphicstest) without GUIslice 
_Adafruit ILI9341
XPT2046_Touchscreen_

•	Select a GUIslice Example Config suitable for your display
_#include "../configs/teensy-adagfx-ili9341-xpt2046.h"_

•	Update GUIslice_config.h to include this Example Config 
_Done._

•	For breakout boards, modify the config pinout to match the wiring connections
_Done._

•	Confirm GUIslice basic graphics diagnostic (ex01_ard_basic) works

**Comment by ImpulseAdventure** [comment, marked as answer]
Hi Josh —

Can you try making a change to the following line in your config file? Change the value from 1 to 0:

`#define GSLC_USE_PROGMEM      0`

thanks

**Reply by JoshBeckmann** [reply]
Many thanks for a useful tool and helpful support. It compiles now without errors.
